src/pages/index.js

```diff
@@ -29,54 +31,185 @@ const enableValidation = (config) => {
 
 enableValidation(config);
 
+//VAR
+
+let cardSection;
+let initialUserData;
+
+//API
+
+export const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authorization: "7dcd9a93-149c-4e56-87db-9285c9177a9e",
+});
+
+api.getServerData().then(([data, cards]) => {
+  initialUserData = new UserInfo(
+    "#profile-name",
+    "#profile-about",
+    "#profile-avatar"
+  );
+  initialUserData.setUserInfo(data);
+
+  cardSection = new Section(
+    {
+      data: cards,
+      renderer: (data) => {
+        const cardElement = createCard(data);
+        cardSection.addItem(cardElement);
+      },
+    },
+    cardListElement
+  );
+  cardSection.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**